frontend/dispatcher.lua

```diff
@@ -121,12 +122,17 @@ local settingsList = {
     ----
 
     -- File browser
+    set_display_mode = {category="string", event="SetDisplayMode", title=_("Set display mode"), args_func=FileManager.getDisplayModeActions, filemanager=true},
```

Comment:
(There are a few other existing args_func and stuff name something**Actions**. Any idea why we named that with **Actions** ? It sounds a bit odd we named these like thins. In general, we name somethingAction the little functions we create to give to `UIManager:scheduleIn(1, do_something_action)`)

Reply:
getDisplayModes?